Makefile

```diff
@@ -415,6 +420,13 @@ debug-server: start-docker ## Compile and start server using delve.
 		-X github.com/mattermost/mattermost-server/v5/model.BuildHashEnterprise=$(BUILD_HASH_ENTERPRISE)\
 		-X github.com/mattermost/mattermost-server/v5/model.BuildEnterpriseReady=$(BUILD_ENTERPRISE_READY)'"
 
+run-server-no-docker: prepackaged-binaries validate-go-version
```

Comment:
Hehe, we actually don't need this. `run-server` does not start docker if you have the `MM_NO_DOCKER` env var set :P

Reply:
Duh, you are so right on this.